pacu/core/models.py

```diff
@@ -190,6 +202,14 @@ def print_all_data_in_session(self) -> None:
         else:
             print('This session has no data.')
 
+    def json_update(self, **kwargs) -> None:
```

Comment:
json_update behaves the same way as the previous session.update, which handled extra json marshalling.